CMakeLists.txt

```diff
@@ -268,6 +268,14 @@ if(WITH_BLUESTORE)
   endif()
 endif()
 
+option(WITH_PMEM_DEVICE "Enable PMEM OSD" OFF)
```

Comment:
could you rename `WITH_PMEM_DEVICE` to something like `WITH_PMEM_OSD` or `WITH_BLUESTORE_PMEM`? as DEVICE is kind of vague in this context.

Reply:
Updated it to WITH_BLUESTORE_PMEM